src/gp-angular.js

```diff
@@ -559,21 +559,22 @@ module.provider('GlobalizationPipelineService', [ function() {
  *  <span gp-translate="aKey"></span>
  *  <span gp-translate="aKey" lang="es"></span>
  */
-module.directive('gpTranslate', ['$log', 'GlobalizationPipelineService', function($log, gp) {
+module.directive('gpTranslate', ['$log', '$interpolate', 'GlobalizationPipelineService', function($log, $interpolate, gp) {

```

Comment:
`$interpolate` is a base Angular service, so no enabling needed.  It just needs to be injected into wherever you're planning to use it as seen here.

Would you like the doc link as a comment above this?
https://docs.angularjs.org/api/ng/service/$interpolate